explore-analyze/alerts-cases/alerts/notifications-domain-allowlist.md

```diff
@@ -6,11 +6,12 @@ applies_to:
   serverless: ga
 products:
   - id: cloud-hosted
+description: Instructions for configuring the domain allowlist that restricts email recipients for alerting notifications.
 ---
 
 # Notifications domain allowlist [organizations-notifications-domain-allowlist]
 
-The notifications domain allowlist restricts the possible recipients for alert emails. {{es}} Watcher and {{kib}} alerting actions send emails only if the recipient domains are included in this allowlist.
+The notifications domain allowlist restricts the possible recipients for alert emails. {{es}} {{watcher}} and {{kib}} alerting actions send emails only if the recipient domains are included in this allowlist. You can configure the allowlist from your organization's Contacts page.
```

Comment:
The advice at the end is an interesting addition. It's generic enough that it could apply broadly. But it also feels outside of the scope of guidance that we normally provide in docs, so I wouldn't include it if I was manually creating this content.